easybuild/easyconfigs/p/Python/Python-3.7.2-fix_faulthandler.patch

```diff
@@ -0,0 +1,16 @@
+diff -Nru Python-3.7.2.original/Modules/faulthandler.c Python-3.7.2/Modules/faulthandler.c
```

Comment:
Please add on top
+ descriptive text of what it solves. Perhaps a link to the issue
+ your name and date

Reply:
Just to be clear, you want this in the patch file itself, not in the PR text, yes?

Reply:
I want it in the patch, so that if someone else looks at it, they understand why
E.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/s/sympy/sympy-1.4_intel_precision.patch